android-app/app/src/main/java/de/dhbw/ka/se/fibo/models/Category.java

```diff
@@ -0,0 +1,54 @@
+package de.dhbw.ka.se.fibo.models;
+
+
+import android.graphics.Color;
+
+import java.util.Random;
+
+
+public enum Category {
+
+
+    RESTAURANT(Color.rgb(216, 49, 91), "Restaurant"), GESUNDHEIT(Color.rgb(89, 178, 34), "Gesundheit"), WOHNEN(Color.rgb(231,74,22), "Wohnen"), HAUSHALT(Color.rgb(33,11,99), "Haushalt"), KULTUR(Color.rgb(53,23,200), "Kultur"), BILDUNG(Color.rgb(234,33,11), "Bildung"), SOZIALLEBEN(Color.rgb(223,22,44), "Sozialleben"), MOBILITÄT(Color.rgb(23,99,22), "Mobilit\u00e4t"), BEKLEIDUNG(Color.rgb(234,200,18), "Bekleidung"), GESCHENK(Color.rgb(66,83,244), "Geschenk"), SONSTIGE(Color.rgb(33,1,9), "Sonstige"), VERSICHERUNG(Color.rgb(23,42,44), "Versicherung");
```

Comment:
Maybe the enum constants should be english

Reply:
… or rather, they should be using a language variable …